### ISSUE TITLE
WARNING: Requirement 'xformers-0.0.23.post1-cp39-cp39-manylinux2014_x86_64.whl' looks like a filename, but the file does not exist ERROR: xformers-0.0.23.post1-cp39-cp39-manylinux2014_x86_64.whl is not a supported wheel on this platform.

### ISSUE THREAD

**Issue by metasl4ve** [pengHTYX/Era3D#34]
Error occurs when running 

`pip install xformers-0.0.23.post1-cp39-cp39-manylinux2014_x86_64.whl`

I'm a 3d artist, so I'm not familiar with this. I checked https://download.pytorch.org/whl/cu118/xformers/ but it seems that wheel is not there. 

xformers-0.0.23.post1**+cu118**-cp39-cp39-manylinux2014_x86_64.whl  seems to be the closest match, please advise where to download the correct whl

**Comment by pengHTYX** [comment]
The file [xformers-0.0.23.post1+cu118-cp39-cp39-manylinux2014_x86_64.whl](https://download.pytorch.org/whl/cu118/xformers-0.0.23.post1%2Bcu118-cp39-cp39-manylinux2014_x86_64.whl#sha256=0905e12df37c9027e8286c1382d459cae675f10a82d9ddb4a9312f93439583cd) should work. Please ensure that the cu118 version matches your CUDA installation.

**Comment by metasl4ve** [comment]
now when I install requirements.txt it gives me 

`ERROR: triton-2.0.0-cp310-cp310-win_amd64.whl is not a supported wheel on this platform.`

Before when installing xformers I ran: `pip install xformers==0.0.23.post1`  to solve my prior issue

which downloaded `xformers-0.0.23.post1-cp39-cp39-win_amd64.whl.metadata`

Is my triton error possibly related to wrong xformer installation?

**Comment by metasl4ve** [comment]
Oh nevermind I see there was an error I missed installing tiny cuda

Collecting git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
  Cloning https://github.com/NVlabs/tiny-cuda-nn/ to c:\users\me\appdata\local\temp\pip-req-build-bwn8omu7
  Running command git clone --filter=blob:none --quiet https://github.com/NVlabs/tiny-cuda-nn/ 'C:\Users\Me\AppData\Local\Temp\pip-req-build-bwn8omu7'
  Resolved https://github.com/NVlabs/tiny-cuda-nn/ to commit b3473c81396fe927293bdfd5a6be32df8769927c
  Running command git submodule update --init --recursive -q
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [23 lines of output]
      C:\Users\Me\AppData\Local\Temp\pip-req-build-bwn8omu7\bindings/torch\setup.py:5: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
        from pkg_resources import parse_version
      'nvcc' is not recognized as an internal or external command,
      operable program or batch file.
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\Me\AppData\Local\Temp\pip-req-build-bwn8omu7\bindings/torch\setup.py", line 187, in <module>
          ext_modules = [make_extension(comp) for comp in compute_capabilities]
        File "C:\Users\Me\AppData\Local\Temp\pip-req-build-bwn8omu7\bindings/torch\setup.py", line 187, in <listcomp>
          ext_modules = [make_extension(comp) for comp in compute_capabilities]
        File "C:\Users\Me\AppData\Local\Temp\pip-req-build-bwn8omu7\bindings/torch\setup.py", line 172, in make_extension
          ext = CUDAExtension(
        File "C:\Users\Me\anaconda3\envs\Era3D\lib\site-packages\torch\utils\cpp_extension.py", line 1076, in CUDAExtension
          library_dirs += library_paths(cuda=True)
        File "C:\Users\Me\anaconda3\envs\Era3D\lib\site-packages\torch\utils\cpp_extension.py", line 1210, in library_paths
          paths.append(_join_cuda_home(lib_dir))
        File "C:\Users\Me\anaconda3\envs\Era3D\lib\site-packages\torch\utils\cpp_extension.py", line 2416, in _join_cuda_home
          raise OSError('CUDA_HOME environment variable is not set. '
      OSError: CUDA_HOME environment variable is not set. Please set it to your CUDA install root.
      Building PyTorch extension for tiny-cuda-nn version 1.7
      Obtained compute capability 89 from PyTorch
      Targeting C++ standard 14
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

**Comment by metasl4ve** [comment]
when installing requirements I get this error: `ERROR: triton-2.0.0-cp310-cp310-win_amd64.whl is not a supported wheel on this platform.`

Confused because the environment is in 3.9. Is this related to xformer, cuda, or torch installation in any way? I was able to install these with no issues

I ended up running `pip install xformers==0.0.23.post1` instead, which installed xformers-0.0.23.post1-cp39-cp39-win_amd64.whl

No cuda support specified, but your installation instructions also don't specify cuda support in the xformer whl, so there shouldn't be an issue, correct?

I later installed cuda `conda install cuda -c nvidia/label/cuda-11.8.0` , after I ran reconstruction installations, no issues.
